Makefile

```diff
@@ -139,6 +139,9 @@ docker-image: clean GIT_VERSION envoy/SOURCE_VERSION
 	$(QUIET)echo "Push like this when ready:"
 	$(QUIET)echo "docker push cilium/cilium:$(DOCKER_IMAGE_TAG)"
 
+docker-image-init:
+	$(QUIET)cd contrib/packaging/docker && docker build -t "cilium/cilium-init:$(UTC_DATE)" -f Dockerfile.init .
```

Comment:
Please create a automatic build repository for `cilium/cilium-init`

Reply:
Will `Dockerfile.init` be read by automated docker hub builds? I am not sure, but it seems like automated build settings only accept a directory which should contain `Dockerfile`. If that's the case, this dockerfile should be moved to a separate dir.

Reply:
@nebril You can specify which dockerfile that dockerhub uses :)